app/controllers/podcast_engagement_controller.rb

```diff
@@ -11,6 +11,7 @@ def update
     respond_to do |format|
       if @podcast.save
         @podcast.publish!
+        @podcast.copy_subscribe_links
```

Comment:
We may eventually want to throw this in a background job, since writing to S3 can be a _little_ bit slow.  But it's just on this 1 form, so could be fine.  Let's find out!